platform/conf/conf.go

```diff
@@ -266,11 +282,18 @@ func (c *Conf) copyKeysIgnitionV1(keys []*agent.Key) {
 }
 
 func (c *Conf) copyKeysIgnitionV2(keys []*agent.Key) {
-	var keyObjs []v2types.SSHAuthorizedKey
+	c.ignitionV2.Passwd.Users = append(c.ignitionV2.Passwd.Users, v2types.User{
```

Comment:
I think we technically want to merge with any existing core user if one exists, otherwise I believe this clobbers existing config.